target/linux/rockchip/rk3399/base-files/etc/board.d/02_network

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+. /lib/functions/uci-defaults.sh
+. /lib/functions.sh
```

Comment:
Use this as a template? Slightly cleaner
https://raw.githubusercontent.com/openwrt/openwrt/master/target/linux/sunxi/base-files/etc/board.d/02_network

Reply:
Done.